userspace/libsinsp/parsers.cpp

```diff
@@ -1319,6 +1319,12 @@ void sinsp_parser::parse_clone_exit(sinsp_evt *evt)
 			// This can happen if the thread table has reached max capacity
 			//
 			ASSERT(false);
+			if (tid == evt->get_tid())
```

Comment:
If this case can actually happen the `ASSERT(false)` above is probably not needed :)

Reply:
I think @LucaGuerra is right!
@gnosek can you open another PR to fix that? Or add a commit to any of your existent PR :laughing:

Reply:
Hey @LucaGuerra 

You're right, but we have seen this comment too late :angel: 

Could you make another PR to fix it, please? :pray:

Reply:
Also added a comment below (not merge blocking!)